oauth2server/apps/accounts/decorators.py

```diff
@@ -148,25 +147,25 @@ def _extract_username(request, **kwargs):
                 user = OAuthUser(email=username, password=password, account_is_verified=False)
                 user.validate_unique()
             except ValidationError:
-                stdlogger.warning("We failed username check for creating a user in the HTTP POST. This user already exists")
+                stdlogger.warning(
+                    "We failed username check for creating a user in the HTTP POST. This user already exists")
                 raise DuplicateUserException
 
         if request.method == 'PUT':
             try:
                 username = request.POST['username']
                 validate_email(username)
                 user = OAuthUser.objects.get(email=username)
-                if user.account_is_locked:
-                    stdlogger.warning("A user trying to update their account has it in a locked state")
-                    raise UserAccountLockedException
             except KeyError:
                 stdlogger.warning("Username is missing from the PUT method")
                 raise UsernameRequiredException()
             except ValidationError:
                 stdlogger.warning("Email failed validation check for validating a user")
                 raise InvalidUserCredentialsException
             except OAuthUser.DoesNotExist:
-                stdlogger.error("Error while updating the User ID. The email: {} does not exist on the OAuth2 server".format(username))
+                stdlogger.error(
+                    "Error while updating the User ID. The email: {} does not exist on the OAuth2 server".format(
```

Comment:
someone was pointing out that email should not be logged ? I find teh information useful but not sure is complaiant with ONS internal rules ? @gavinleeedwards